.github/workflows/ci.yml

```diff
@@ -174,8 +174,6 @@ jobs:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@main
     with:
       sdist: false
-      test_extras: 'test'
-      test_command: 'pytest -p no:warnings --doctest-rst -m "not mpl_image_compare" --pyargs sunpy'
```

Comment:
One of the limitations of the OpenAstronomy workflow is that these two options cannot be specified in `pyproject.toml`. [These four cibuildwheel options](https://github.com/OpenAstronomy/github-actions-workflows/blob/bb12d5420b890a2102eea80fbfbda86e626baec4/.github/workflows/publish.yml#L165-L168) are set as env vars in the workflow, which override the `pyproject.toml` config. You can see [in the build logs](https://github.com/sunpy/sunpy/actions/runs/4779231611/jobs/8496280260#step:6:72) that it skipped testing as the env var was an empty string.

Reply:
😆 snap, I just opened an issues about this over there